terralab/config.py

```diff
@@ -26,11 +26,16 @@ def __init__(self, config_file=".terralab-cli-config", package="terralab"):
         self.client_info = OAuth2ClientInfo.from_oidc_endpoint(
             self.config["OAUTH_OPENID_CONFIGURATION_URI"],
             client_id=self.config["OAUTH_CLIENT_ID"],
-            scopes=[f"openid+email+profile+{self.config['OAUTH_CLIENT_ID']}"],
+            # including the offline_access scope is how we request a refresh token
+            scopes=[f"offline_access+email+profile+{self.config['OAUTH_CLIENT_ID']}"],
```

Comment:
is "openid" no longer needed in the scopes?

Reply:
that's right, no longer needed